common/api-review/storage.api.md

```diff
@@ -217,8 +215,71 @@ export interface SettableMetadata {
 }
 
 // @public
-export interface StorageError extends FirebaseError {
-    serverResponse: string | null;
+export class StorageError extends FirebaseError {
+    constructor(code: StorageErrorCode, message: string, status_?: number);
+    _codeEquals(code: StorageErrorCode): boolean;
+    customData: {
+        serverResponse: string | null;
+    };
+    get serverResponse(): null | string;
+    set serverResponse(serverResponse: string | null);
+    // (undocumented)
+    get status(): number;
+    set status(status: number);
+    }
+
+// @public
+export const enum StorageErrorCode {
```

Comment:
Sorry, wasn't paying attention earlier to a new enum being exported as well, I think there are some problems with exporting a `const enum` (for a certain set of users) so I think this needs to be changed to just `enum` here https://github.com/firebase/firebase-js-sdk/blob/68369740aa8802ea8391432f10f9baa4abb84aa0/packages/storage/src/implementation/error.ts#L88
Background on const enum issues https://ncjamieson.com/dont-export-const-enums/